lefthook.yaml

```diff
@@ -87,6 +87,26 @@ pre-push:
           exit 1
         fi
 
+tests:
+  follow: true
```

Comment:
@Nadav-B I added this config to log the test output during execution. Of course the lines of all three processes will be mixed, but still some real-time feedback. Let me know what you think!